templates/blog/macros.html

```diff
@@ -2,7 +2,7 @@
   <ul>
   {% for article in articles %}
   {% set url = url_for_article(article) %}
-  <li class=articles>
+  <li class=article>
```

Comment:
Ok I agree with you, I read a lot of article about it (yes I was thinking about it during my trip ^^).
I will change all the structure :)